tests/test_utilities.py

```diff
@@ -96,3 +99,18 @@ def test_hss_solver_config_override(self):
             self.assertTrue(FeatureFlags.hss_solver_config_override)
         with isolated_environ(remove=('DWAVE_FEATURE_FLAGS',)):
             self.assertFalse(FeatureFlags.hss_solver_config_override)
+
+
+class TestScheduleWithAnnealOffset(unittest.TestCase):
+    def test_doc_file(self):
+
+        csv_file =  os.path.join(
+            os.path.dirname( __file__ ),
+            '..',
+            'docs',
+            '_static',
+            'advantage_system4_1_annealing_schedule_standard.csv')
+        schedule = np.loadtxt(csv_file, delimiter=",", skiprows=1)
+        schedule_offset = schedule_with_anneal_offset(csv_file, 0.2)
+
+        np.testing.assert_array_equal(schedule_offset[:, 0], schedule[:, 0])
```

Comment:
This is not testing much, just that the `s` axis didn't change.

We should test that interpolation actually works.

Reply:
Added unittesting of the interpolation